packages/store/addon/-private/system/store.ts

```diff
@@ -703,12 +705,12 @@ const Store = Service.extend({
     );
     assert(badIdFormatAssertion, (typeof id === 'string' && id.length > 0) || (typeof id === 'number' && !isNaN(id)));
 
-    let normalizedModelName = normalizeModelName(modelName);
-
-    let internalModel = this._internalModelForId(normalizedModelName, id);
+    const normalizedModelName = normalizeModelName(modelName);
```

Comment:
I recall a discussion in which we decided to only use `const` for constants, and not inside methods.